views/components/flexible-content/macro.njk

```diff
@@ -1,39 +1,56 @@
 {% from "components/callout/macro.njk" import callout %}
 {% from "components/media-aside/macro.njk" import mediaAside %}
 
+{% macro flexibleContentPart(part) %}
```

Comment:
Not the `mediaAsideCounter` logic probably doesn't work here now this has been extracted out.

Reply:
Have fixed this now using the nifty `cycler` tool in Nunjucks, which only calls its next parameter when `next()` is triggered.

![image](https://user-images.githubusercontent.com/394376/55337457-a1ecf780-5496-11e9-8f56-9fdec08eca99.png)

Reply:
Nice